war/pom.xml

```diff
@@ -103,7 +103,7 @@ THE SOFTWARE.
       -->
       <groupId>org.jenkins-ci</groupId>
       <artifactId>winstone</artifactId>
-      <version>5.11.1</version>
+      <version>5.12</version>
```

Comment:
Why are we backporting this upgrade? Does this solve any user-visible bug?

Reply:
Good point, it does not. Though it gets rid of the downgraded Jetty inside that we have used to avoid a now fixed regression.

(Winsone 5.10 and 5.11.1 both contain Jetty 9.4.30.v20200611, while 5.12 have 9.4.33.v20201020.)

Reply:
To justify the Jetty upgrade, please see https://github.com/eclipse/jetty.project/security/advisories/GHSA-g3wg-6mcf-8jj6 . "Local Temp Directory Hijacking Vulnerability" for temporary directories on some of the operating systems. We have done investigation in the Security team, and so far we do not see a use-case where Jenkins would be affected. At the same time, it would make sense to upgrade. Jetty was updated in 2.264, and I do not see any regressions which would be caused by this upgrade.

Reply:
I think this is a reasonable backport. We missed the weekly by 1 release, nothing has come up since, and otherwise we'll be bombarded with security scan output for the next 3 months.

Reply:
OK, I was missing this context but agree now.